core-bundle/src/Command/MigrateCommand.php

```diff
@@ -485,6 +485,10 @@ private function compileTableWarnings(Table $table): array
     {
         $warnings = [];
 
+        if ($table->hasOption('engine') && 'innodb' !== strtolower($table->getOption('engine'))) {
```

Comment:
Can the 'engine' option be empty and we're still dealing with anything else than innodb? 
If so should we turn the condition around?

```suggestion
        if (!$table->hasOption('engine') || 'innodb' === strtolower($table->getOption('engine'))) {
```

Reply:
AFAIK the engine defaults to `InnoDB`